jedi/api/project.py

```diff
@@ -366,8 +366,11 @@ def __repr__(self):
 
 def _is_potential_project(path):
     for name in _CONTAINS_POTENTIAL_PROJECT:
-        if path.joinpath(name).exists():
-            return True
+        try:
+            if path.joinpath(name).exists():
+                return True
+        except PermissionError:
```

Comment:
Just open for discussion: Shouldn't we use OSError here? I'm pretty sure that we do not care here if we can read it. If it's not readable, it should probably be ignored.

Reply:
I just replaced `PermissionError` by `OSError`